src/flint/routers/api.py

```diff
@@ -55,11 +61,41 @@ async def chat(
         user = await sync_to_async(user.get)()
     uuid = user.khojuser.uuid
 
+    # Set user message to the body of the request
+    user_message = Body
```

Comment:
IMO we should move the message processing logic into an async method to avoid having multiple open threads with Twilio waiting for an API response.